Duality/Resources/Font.cs

```diff
@@ -107,14 +107,19 @@ public struct GlyphData
 			/// The glyphs kerning samples to the right.
 			/// </summary>
 			public int[] KerningSamplesRight;
+			/// <summary>
+			/// The glyphs Y offset when rendering it.
+			/// </summary>
+			public int OffsetY;

```

Comment:
Please move this field up, so it's directly under `OffsetX`, just for reading flow reasons. Don't worry about binary compatibility, the `BinarySerializer` saves enough metadata to deal with this properly.